dist/t/spec/features/0040_package_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+
+RSpec.describe "Package" do
+
+  it "should be able to create new" do
+    login('Admin','opensuse')
+    within("div#subheader") do
+      click_link('Home Project')
+    end
+    expect(current_path).to eq '/project/show/home:Admin'
+    click_link('Create package')
+    fill_in 'name', with: 'ctris'
+    fill_in 'title', with: 'ctris'
+    fill_in 'description', with: 'ctris'
+    click_button('Save changes')
+    expect(page).to have_content("Package 'ctris' was created successfully")
+
+    logout
```

Comment:
I would put this line in an `after` block. And remove all the `logout` lines in the file.